### PR TITLE
Add page on how to retire a gem

### PR DESCRIPTION
**Pull request by MuriloDalRi**
https://trello.com/c/1IWOqO4r/2980-decide-how-to-properly-archive-a-gem-and-document-it-2

### CODE REVIEW

File: source/manual/retiring-a-gem.html.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+Mark the application as `retired` in [govuk-developer-docs][https://github.com/alphagov/govuk-developer-docs].
+
+## 4. Yank the gem
+
+Once the gem has been archived you may want to [yank it from Rubygems](https://guides.rubygems.org/removing-a-published-gem/) if it was published there. You can find the Rubygems credentials in [govuk-secrets](https://github.com/alphagov/govuk-secrets).
+
+## 5. Update the dependency management spreadsheet
+
+Remove the gem from the [GOV.UK dependency management spreadsheet](https://docs.google.com/spreadsheets/d/137KZhjctJ8qTKYPnq2QNVkyoIC6ok7KA2G1vErNC6Oo/edit#gid=1160366178)
```
Comment (ChrisBAshton) [inline source/manual/retiring-a-gem.html.md:45]: We can delete this I think - it would automatically happen next time we run "Update Data".

**Review by ChrisBAshton — COMMENTED** [review]
Thanks Murilo, a good first draft 👍 a few comments below.

File: source/manual/retiring-a-gem.html.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+Go into the repository settings in GitHub, and
+[archive the repo](https://github.com/blog/2460-archiving-repositories).
+
+## 3. Remove references and update docs
+
+Do a [code search on GitHub][https://github.com/search?q=org%3Aalphagov+panopticon&type=Code] to find any references to the application
+and update or remove them.
+
+Mark the application as `retired` in [govuk-developer-docs][https://github.com/alphagov/govuk-developer-docs].
```
Comment (nicholsj) [inline source/manual/retiring-a-gem.html.md:37]:
```suggestion
Mark the gem as `retired` in [govuk-developer-docs][https://github.com/alphagov/govuk-developer-docs].
```

File: source/manual/retiring-a-gem.html.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+When the gem's repo is ready to be archived, push a commit explaining the reasons for archiving it.
+
+[Example](https://github.com/alphagov/govuk_taxonomy_helpers/pull/27)
+
+Go into the repository settings in GitHub, and
+[archive the repo](https://github.com/blog/2460-archiving-repositories).
+
+## 3. Remove references and update docs
+
+Do a [code search on GitHub][https://github.com/search?q=org%3Aalphagov+panopticon&type=Code] to find any references to the application
```
Comment (nicholsj) [inline source/manual/retiring-a-gem.html.md:34]:
```suggestion
Do a [code search on GitHub][https://github.com/search?q=org%3Aalphagov+panopticon&type=Code] to find any references to the gem
```

**Comment by MuriloDalRi** [comment]
Thanks for all the comments, @ChrisBAshton and @nicholsj , I've made the suggested changes.

**Review by ChrisBAshton — APPROVED** [review]
👍

**Pull request merged**